ui-tests/src/test/java/io/syndesis/qe/steps/CommonSteps.java

```diff
@@ -67,6 +68,171 @@ public void resetState() {
 	@Then("^\"(\\w+)\" is presented with the Syndesis home page.")
 	public void checkHomePageVisibility(String username) {
 		SyndesisRootPage syndesisRootPage = new SyndesisRootPage();
-		syndesisRootPage.getRootElement().shouldBe(Condition.visible);
+		syndesisRootPage.getRootElement().shouldBe(visible);
+	}
+
+	@Then("^(\\w+)? is presented with the \"([^\"]*)\" link*$/")
+	public void validateLink(String alias, String linkTitle) {
+		new SyndesisRootPage().getLink(linkTitle).shouldBe(visible);
+	}
+
+	@When("clicks? on the \"([^\"]*)\" button.*$/")
+	public void clickOnButton(String buttonTitle, String callback) {
```

Comment:
We need to get rid off those `callback` params. Those aren't needed anymore. And why is there question mark in `@When("clicks? on the \"([^\"]*)\" button.*$/")`

Reply:
I Agree, the next iteration will fix this. I just wanted to add the methods as soon as possible without investigating how the regexp should look like. We can use them in our tests meanwhile.

Reply:
So should I merge this PR then? Just say a word... :)